Formula/ttygif.rb

```diff
@@ -0,0 +1,18 @@
+class Ttygif < Formula
+  desc "Converts a ttyrec file into gif files"
+  homepage "https://github.com/icholy/ttygif"
+  url "https://github.com/icholy/ttygif/archive/1.3.0.tar.gz"
+  sha256 "18ed967723bacdc1ced6d8ff86bcb3385d940863b8bcb4d3f6b31344f035d7a9"
+
+  depends_on "imagemagick"
+  depends_on "ttyrec"
+
+  def install
+    system "make"
+    bin.install("ttygif")
+  end
+
+  test do
+    system "true"
```

Comment:
You can use an existing file; you don’t have to record it during the test. See e.g. https://github.com/Homebrew/homebrew-core/blob/dae6d3ebeb1e9f269092adf6677aa0922d3680b8/Formula/ttyrec.rb#L16